src/widgets/snippetpanel.cpp

```diff
@@ -76,9 +76,9 @@ void SnippetPanel::setupTitleBar(const QString &p_title, QWidget *p_parent)
 
     auto showAct = m_titleBar->addMenuAction(tr("Show Built-In Snippets"),
                                              m_titleBar,
-                                             [this](bool p_checked) {
-                                                 ConfigMgr::getInst().getWidgetConfig().setSnippetPanelBuiltInSnippetsVisible(p_checked);
-                                                 setBuiltInSnippetsVisible(p_checked);
+                                             [this](bool m_builtInSnippetsVisible) {
```

Comment:
Revert the change on this file. `p_checked` is a parameter will be passed to the lamda.